src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -140,6 +140,32 @@
  * </li>
  * </ul>
  * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="ImportOrder"/&gt;
+ * </pre>
+ * <p>
+ * Example of Import order:
+ * </p>
+ * <pre>
+ * import java.io.IOException; //OK
+ * import java.net.URL; //OK
+ * import java.security.KeyManagementException; //OK
+ *
+ * import javax.net.ssl.SSLContext; //OK
+ * import javax.net.ssl.TrustManager; //OK
+ * import javax.net.ssl.X509TrustManager; //OK
+ *
+ * import org.apache.http.conn.ClientConnectionManager; //OK
+ * import org.apache.http.conn.scheme.Scheme; //OK
+ *
+ * import com.neurologic.http.HttpClient; //OK
+ * import com.neurologic.http.impl.ApacheHttpClient; //OK
+ *
+ * public class SomeClass { ... }
```

Comment:
It is a good idea use compilable code in examples.  A good workflow that I use when updating check documentation is very similar to creating bug reports:

1. Create the configuration
2. Create the the file to be checked
3. Make sure code compiles
4. Run checkstyle on the file
5. place "// ok" and "// violation" comments where appropriate
6. Paste into documentation

This will help you to learn Checkstyle and develop a good work flow.

Reply:
how do i check a file?

Reply:
> how do i check a file?
https://checkstyle.sourceforge.io/cmdline.html

Reply:
that javac code